ofj-template.tex

```diff
@@ -69,7 +69,7 @@
 \let\paragraph\undefined
 
 % Copyright info - will be updated by the editor
-\copyrightinfo{2022}{The Authors. This work is an open access article under the {\color{blue}{\href{https://creativecommons.org/licenses/by-sa/4.0/}{CC BY-SA 4.0}}} license}
+\copyrightinfo{2024}{The Authors. This work is an open access article under the {\color{blue}{\href{https://creativecommons.org/licenses/by-sa/4.0/}{CC BY-SA 4.0}}} license}
```

Comment:
```suggestion
\copyrightinfo{2025}{The Authors. This work is an open access article under the {\color{blue}{\href{https://creativecommons.org/licenses/by-sa/4.0/}{CC BY-SA 4.0}}} license}
```

Reply:
Thank you, this update is also relevant.